packages/nextra/src/index.ts

```diff
@@ -0,0 +1,69 @@
+import { NextraPlugin, pageMapCache } from './plugin'
+import { MARKDOWN_EXTENSION_REGEX } from './constants'
+import { LoaderOptions, Nextra } from './types'
+
+const DEFAULT_EXTENSIONS = ['js', 'jsx', 'ts', 'tsx'] as const
+const MARKDOWN_EXTENSIONS = ['md', 'mdx'] as const
+
+const nextra: Nextra = (...args) =>
+  function withNextra(nextConfig = {}) {
+    const nextraConfig =
+      typeof args[0] === 'string'
+        ? {
+            theme: args[0],
+            themeConfig: args[1] as string
+          }
+        : args[0]
+
+    const nextraPlugin = new NextraPlugin(nextraConfig)
+    const { i18n, pageExtensions = DEFAULT_EXTENSIONS } = nextConfig
+
+    if (i18n?.locales) {
+      console.log(
+        '[nextra] You have Next.js i18n enabled, read here https://nextjs.org/docs/advanced-features/i18n-routing for the docs.'
```

Comment:
replaced todo link

Reply:
Sounds good! When Nextra has a good i18n docs page we should link there